src/app/_components/product.tsx

```diff
@@ -0,0 +1,221 @@
+                {name}
+              </h1>
+            </div>
+            <div className="h-6 w-[1.7rem] bg-[url('/card/favorite-product.svg')]"></div>
+          </div>
+          <div className="flex items-center gap-7">
+            <div className="flex items-center gap-1">
+              <div className="-mt-[0.15rem] h-4 w-4 bg-[url('/card/star-product.svg')]"></div>
+              <p className="cursor-pointer bg-custom-gradient bg-clip-text text-sm">
+                {calification}
+              </p>
+            </div>
+            <p className="bg-clip-text text-sm text-blue-700">
+              Ver calificaciones
+            </p>
+          </div>
+          <div className="mb-8 flex gap-0">
+            <button
+              onClick={decreaseQuantity}
+              className="rounded-l-md bg-gray-200 p-2 hover:bg-gray-300 focus:outline-none focus:ring-2 focus:ring-gray-400 focus:ring-opacity-50"
+            >
+              -
+            </button>
+            <div className="flex w-12 items-center justify-center border-y border-gray-200 text-center">
+              {quantity}
+            </div>
+            <button
+              onClick={increaseQuantity}
+              className="rounded-r-md bg-gray-200 p-2 text-black hover:bg-gray-300 focus:outline-none focus:ring-2 focus:ring-gray-400 focus:ring-opacity-50"
+            >
+              +
+            </button>
+          </div>
+        </div>
+        <div className="mb-6 space-y-2">
+          <h2 className="g-clip-text text-xl ">Personaliza tu box</h2>
+          {personalize?.map((option, index) => (
+            <div key={index} className="flex items-center space-x-2">
+              <Switch />
+              <label htmlFor={`option-${index}`} className="text-gray-800">
+                {option}
+              </label>
+            </div>
+          ))}
+          <div className="flex h-10 w-56 flex-col items-start">
+            <p className="text-right text-sm font-light leading-6 tracking-tighter text-gray-400">
+              ¿Necesita más personalización?
+            </p>
+            <p className="cursor-pointer bg-gradient-to-r from-blue-700 to-blue-300 bg-clip-text text-right text-sm font-light leading-6 tracking-tighter text-transparent">
+              Hablar con un ejecutivo
+            </p>
+          </div>
+        </div>
+
+        <div className="flex flex-col">
+          <div className="mb-10 mt-10 flex flex-row justify-between text-2xl font-medium text-customGray">
+            TOTAL
+            <div>
+              ${price}
+              <span className="text-text">+ IVA</span>
+            </div>
+          </div>
+          <Button className={buttonVariants({ variant: "primary" })}>
+            Comprar
+          </Button>
+          <p className="cursor-pointer bg-custom-gradient bg-clip-text text-center text-sm font-light leading-6 tracking-tighter text-transparent">
+            Imprimir presupuesto
```

Comment:
Let's not do it, but let's keep it in mind. As I mentioned previously, sometimes is good to try to move props that are inherited to tags that are above on the tree structure, so then the styles are inherited. For example here I'm seeing that tracking-tighter is being repeated everywhere. Also, remember always checking if something is already the default value por a property, I'm not sure in this case, but tracking-tighter (it doesn't look like it is judging from the name) might be the default value, so let's not always copy al the styles directly from Figma.